pkg/common/crypto/default.go

```diff
@@ -0,0 +1,47 @@
+package crypto
+
+import (
+	"errors"
+
+	"github.com/hyperledger/aries-framework-go/pkg/framework/provider/crypto"
+	spicrypto "github.com/hyperledger/aries-framework-go/pkg/framework/spi/crypto"
+)
+
+// MainCrypto is the main crypto operations provider to the Aries Framework
+type MainCrypto struct {
+	c *crypto.TinkCrypto
+}
+
+// New return new instance of KMS implementation
+func New(cr spicrypto.Crypto) (FrameworkCrypto, error) {
+	tc, ok := (cr).(*crypto.TinkCrypto)
+	if !ok {
+		return nil, errors.New("bad Crypto type")
+	}
```

Comment:
why do we need a specific type?

Reply:
```
func New(cr spicrypto.Crypto) FrameworkCrypto {
	return &MainCrypto{c: tc}
}
```

Reply:
default implementation has been removed.. Crypto is created via context creating in the framework which makes this implementation obsolete.